src/js/modules/infragistics.ui.editors.js

```diff
@@ -1592,7 +1593,22 @@
 				$(".selector").%%WidgetName%%("option", "suppressNotifications", true);
 			```
 			*/
-			suppressNotifications: false
+			suppressNotifications: false,
+			/* type="bool" Suppress on-screen keyboard from showing when the dropdown button is clicked and the list is opened.

```

Comment:
@IG-Stanimir-Dimitrov I don't get the `(touch devices only)` bit though - that was my point for the note - there's no mobile device detection built in and I wanted that to be very clear.
The option could run just as well on non-touch desktops if someone prefers the "open dropdown without focus" behavior.
```
type="bool" Gets/Sets whether the onscreen keyboard (on touch devices) should be shown when the dropdown button is clicked . This option prevents initial focus or removes it when the drop button is clicked.

Note: The option does not perform device detection so its behavior is always active if enabled.
```
?
the "clicked" wording is also not super correct